validator/lib/readTileset.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
`Running gzip
Compressing files...
Unhandled rejection Error: EISDIR: illegal operation on a directory, open 'output/TilesetOfTilesets-gzipped'
    at Error (native)`

This is what I receive on trying to gzip the tileset. Any advice on how to fix this?